storage_s3/indico_storage_s3/task.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import re
+from datetime import datetime
+
+from celery.schedules import crontab
+from dateutil.relativedelta import relativedelta
+
+from indico.core.celery import celery
+from indico.core.storage.backend import get_storage, get_storage_backends
+
+from plugin import S3Storage, S3StoragePlugin
+
+
+@celery.periodic_task(run_every=crontab(minute=0, hour=1))
+def create_bucket():
+    storages = get_storage_backends()
+    for key, storage_class in storages.iteritems():
+        if storage_class == S3Storage:
+            S3StoragePlugin.logger.info('S3 storage instance found: %s' % key)
```

Comment:
don't interpolate values into logged strings; that doesn't work well with sentry (which groups by log string). instead, pass them as posargs:

    S3StoragePlugin.logger.info('S3 storage instance found: %s', key)

Reply:
Also, this particular message should be `debug` if you want to keep it. It's just noise and nothing one would need/want in a non-debug log.